istio/README.md

```diff
@@ -112,6 +112,112 @@ See [service_checks.json][16] for a list of service checks provided by this inte
 
 ## Troubleshooting
 
+### Using Openmetrics Integration
+
+If Istio proxy sidecar injection is enabled, monitoring other Prometheus metrics via the [Openmetrics integration][20] can result in high custom metrics usage and duplicated metric collection.
+
+To ensure that your Openmetrics configuration is not redundantly collecting metrics, either:
+
+1. Use specific metric matching in the `metrics` configuration option
+2. If you use `*` value for `metrics`, then at minimum, consider including the following optional options to exclude metrics already collected by istio and Envoy inegrations.
+
+#### Openmetrics V2 configuration
+
+```yaml
+## Every instance is scheduled independent of the others.
+#
+instances:
+
+  -
+    ## @param openmetrics_endpoint - string - optional
+    ## The URL exposing metrics in the OpenMetrics format.
+    #
+    openmetrics_endpoint: <OPENMETRICS_ENDPOINT>
+
+    ## @param metrics - (list of string or mapping) - required
+    ## This list defines which metrics to collect from the `openmetrics_endpoint`.
+    ## Metrics may be defined in 3 ways:
+    ##
+    ## 1. If the item is a string, then it represents the exposed metric name, and
+    ##    the sent metric name will be identical. For example:
+    ##
+    ##      metrics:
+    ##      - <METRIC_1>
+    ##      - <METRIC_2>
+    ## 2. If the item is a mapping, then the keys represent the exposed metric names.
+    ##
+    ##      a. If a value is a string, then it represents the sent metric name. For example:
+    ##
+    ##           metrics:
+    ##           - <EXPOSED_METRIC_1>: <SENT_METRIC_1>
+    ##           - <EXPOSED_METRIC_2>: <SENT_METRIC_2>
+    ##      b. If a value is a mapping, then it must have a `name` and/or `type` key.
+    ##         The `name` represents the sent metric name, and the `type` represents how
+    ##         the metric should be handled, overriding any type information the endpoint
+    ##         may provide. For example:
+    ##
+    ##           metrics:
+    ##           - <EXPOSED_METRIC_1>:
+    ##               name: <SENT_METRIC_1>
+    ##               type: <METRIC_TYPE_1>
+    ##           - <EXPOSED_METRIC_2>:
+    ##               name: <SENT_METRIC_2>
+    ##               type: <METRIC_TYPE_2>
+    ##
+    ##         The supported native types are `gauge`, `counter`, `histogram`, and `summary`.
+    ##
+    ## Regular expressions may be used to match the exposed metric names, for example:
+    ##
+    ##   metrics:
+    ##   - ^network_(ingress|egress)_.+
+    ##   - .+:
+    ##       type: gauge
+    #
```

Comment:
I think we can leave out the comments, since they can find it in `conf.yaml.example`. WDYT?

Reply:
hmm, my concern with that is it will be confusing if they erroneously assume this to be for Istio conf.yaml.example. I wanted users to know exactly what they're enabling.